examples/from_source/index.bzl

```diff
@@ -1,6 +1,6 @@
 """Example package for generating stardoc from rules_nodejs at source"""
 
-load("@build_bazel_rules_nodejs//packages/typescript:index.bzl", "ts_library")
+load("@build_bazel_rules_nodejs//packages/typescript:index.bzl", "ts_project")
 
 def custom_ts_library(name, deps = [], **kwargs):
```

Comment:
custom_ts_project instead or refactor and get rid of the macro?

Reply:
the macro is here to give coverage for the stardoc use case. I'll just fix the naming.